src/targets/gpu/include/migraphx/gpu/hipblaslt.hpp

```diff
@@ -25,22 +25,46 @@
 #define MIGRAPHX_GUARD_MIGRAPHLIB_HIPBLASLT_HPP
 #include <migraphx/manage_ptr.hpp>
 #include <migraphx/gpu/config.hpp>
+#include <migraphx/errors.hpp>
 #if MIGRAPHX_USE_HIPBLASLT
 #include <hipblaslt/hipblaslt.h>
 #define HIPBLASLT_WORKSPACE_SIZE (2 * 128 * 1024 * 1024)
-#ifndef CHECK_HIPBLAS_ERROR
-#define CHECK_HIPBLAS_ERROR(error)                    \
-    if(error != HIPBLAS_STATUS_SUCCESS)               \
-    {                                                 \
-        fprintf(stderr,                               \
-                "hipBLAS error: '%s'(%d) at %s:%d\n", \
-                hipblasStatusToString(error),         \
-                error,                                \
-                __FILE__,                             \
-                __LINE__);                            \
-        exit(EXIT_FAILURE);                           \
+#ifndef HIPBLASLT_INVOKE_FUNCTION
+#define HIPBLASLT_INVOKE_FUNCTION
```

Comment:
remove these ifdefs

Reply:
Done. I have also addressed the other comments that we discussed, in person, on Monday. Can you please also take a look at the last few commits to see if the comments have been addressed correctly? Thanks a lot for the help!